### PR TITLE
Bump Traefik to v2.4.7

### PR DESCRIPTION
**Pull request by jbdoumenjou**
Hello,

This PR updates Traefik to [v2.4.7](https://github.com/traefik/traefik/releases/tag/v2.4.7).

/cc @ldez @rtribotte @SantoDE

Cheers

### CODE REVIEW

**Review by ldez — APPROVED** [review]
LGTM

**Review by rtribotte — APPROVED** [review]
LGTM

**Comment by tianon** [comment]
(FWIW, we do not need a LGTM from every image maintainer -- we'll typically only hold things up if they're proposed by a non-maintainer, we notice something odd in the review, or another maintainer expresses explicit dissent / request for pause :+1:)

**Pull request merged**
